develop/parachain-devs/parachain-maintenance/runtime-upgrades.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Runtime Upgrades
+description: Explains how runtime versioning and storage migration support forkless upgrades for Polkadot SDK-based networks and they factor in to chain upgrades.
+---
+
+# Runtime Upgrades
+
+## Introduction
+
+One of the defining features of Polkadot SDK-based blockchains is the ability to perform forkless runtime upgrades. Unlike traditional blockchains, which require hard forks and node coordination for upgrades, Polkadot networks enable seamless updates without network disruption. Forkless upgrades are achieved through the use of WebAssembly (Wasm) runtimes stored on-chain, which can be securely swapped in and upgraded as part of the blockchain's state. By leveraging decentralized consensus, runtime updates can be made trustlessly, ensuring continuous improvement and evolution without halting operations. This guide explores how runtime versioning, Wasm deployment, and storage migrations facilitate these upgrades.
+
```

Comment:
```suggestion

## Process Overview

```

Reply:
I don't know if that's the best title, but I feel like this doesn't flow in the introduction. So we can either create a section, or find a way to better embed into the intro

Reply:
I reworked that section a little bit. wdyt?